### ISSUE TITLE
Remove master branch to avoid future confusion

### ISSUE THREAD

**Issue by vorburger** [m2e-code-quality/m2e-code-quality#181]
Does any one, in particular @erwint, have any objection if we just completely delete the master branch?

It seems to me that after #123 and #143 and #176 (#179) having a master branch is more confusing than useful, so to make it easier for future contributors to find their way around this project, why not just delete it? (Just for history, it's currently at d8d98aee7da06c0fbf03beb068a3bd461ac6caf0 from today, previous was 5f21765d9cc28298a3b85e293715ef4e57c152a6 from 2 years, 3 months ago).

We could wait until https://git.eclipse.org/r/#/c/132982/ (re. #157) is merged.

**Comment by erwint** [comment]
Yes, this is quite confusing. We *could* merge develop to master when releasing, so it would be the latest released commit, but I'm fine with completely dropping it, too. Anyone else with an opion on this?

**Comment by erwint** [comment]
PR #185 would keep the master branch and merge with develop when releasing. So it would always have the latest released version (gitflow style). I noticed that it was previously also used in that style.

Of course, if we want to drop master that can be easily removed from that PR
